actions/helm/test-chart/action.yml

```
@@ -24,6 +24,17 @@ inputs:
       List of Helm repositories to add before testing charts.
       See <https://helm.sh/docs/helm/helm_repo_add/>.
     required: false
+  docker-registry:
+    description: "Docker registry"
+    required: false
+    default: "ghcr.io"
+  docker-username:
+    description: "Docker username"
+    required: false
+  docker-password:
+    description: "Docker password"
+    required: false
```

We used to avoid useing "docker" to be agnostic, the others actions are using "OCI":

```suggestion
  oci-registry:
    description: "OCI registry where to pull and push images"
    default: "ghcr.io"
    required: false
  oci-registry-username:
    description: |
      Username used to log against the OCI registry.
      See <https://github.com/docker/login-action#usage>.
    default: ${{ github.repository_owner }}
    required: false
  oci-registry-password:
    description: |
      Password or personal access token used to log against the OCI registry.
      Can be passed in using "secrets.GITHUB_TOKEN".
      See <https://github.com/docker/login-action#usage>.
    default: ${{ github.token }}
    required: false
```